linter_base/src/lints.scala

```diff
@@ -345,7 +472,8 @@ object Unfinished_Proof
     "Consider finishing the proof.",
     "unfinished_proof",
     List("sorry", "oops", "\\<proof>"),
```

Comment:
Does not render properly

Reply:
I think it does, it's supposed to render as `\<proof>`